### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by caseydavenport**
## Description

See here: https://github.com/caseydavenport/calico/tree/readme-updates

Cleans up the README a bit in order to direct visitors to the right place.

Would like to eventually write up a devel guide similar to this one: https://github.com/kubernetes/community/blob/master/contributors/devel/README.md

But can only do so much at once :)

### CODE REVIEW

**Comment by caseydavenport** [comment]
Looks like potentially some overlap with https://github.com/projectcalico/calico/pull/1059/files

@ozdanborne @emanic could you take look at this and see if it's in the right direction?  If it is I can rebase and update, otherwise I can close.

**Comment by emanic** [comment]
@caseydavenport I was mainly focused on updating the documentation guidelines and did not modify the code contribution guidelines except to put them as a readme at the root of the repo with a descriptive name to match the documentation readme. Your changes look good to me except perhaps the file name. Also, I think we might need different guidelines for code and documentation because of the intricacies of building the site and other differences. But let me know if you'd like to discuss this.

**Comment by caseydavenport** [comment]
@emanic @ozdanborne This guy is ready for a review if you're available!

File: README.md [hunk 1]
```diff
@@ -4,48 +4,28 @@
 [![](https://badge.imagelayers.io/calico/node:latest.svg)](https://imagelayers.io/?images=calico/node:latest)
 
 # Calico
+<img src="http://docs.projectcalico.org/images/felix.png" width="100" height="100">
 
-This repository contains the source code for [Project Calico](https://www.projectcalico.org/)'s documentation and demos as well as the source for the `calico/node` container.
+Calico is an open source system enabling cloud native application connectivity and policy.  Calico integrates with major orchestration
```
Comment (emanic) [inline README.md]: Single space between sentences.

**Comment by emanic** [comment]
LGTM 👍

File: README.md (hunk 1 above)
Comment (caseydavenport) [reply README.md]: d'oh, good spot.

File: README.md [hunk 2]
```diff
@@ -4,48 +4,28 @@
...
-<blockquote>
-Note that the README in this repo is targeted at Calico docs contributors.
-<h1>Documentation for Calico users is here:<br><a href="http://docs.projectcalico.org">http://docs.projectcalico.org</a></h1>
-</blockquote>
+Calico is a [Tigera](https://www.tigera.io/) open source project, and is primarily maintained by the Tigera team.
 
+## Get Started Using Calico
 
-For information on `calico/node`, see the [documentation on calico/node architecture](https://docs.projectcalico.org/master/reference/architecture/components).
+For users who want to learn more about the project or get started with Calico, see the documentation on [docs.projectcalico.org](docs.projectcalico.org).
```
Comment (ozdanborne) [inline README.md]: this link needs `https://`, otherwise it think that is the name of a doc

File: README.md [hunk 3]
```diff
@@ -4,48 +4,28 @@
...
 
-### Developing
+## Get Started Developing Calico
 
-Print useful actions with `make help`.
+- [Contribution guidlines](CONTRIBUTING_CODE.md): submitting patches to Calico and contribution workflow.
+- [Building Calico](BUILDING_CALICO.md): building Calico from source and developer workflow.
+- [Contributing to the Calico documentation](CONTRIBUTING_DOCS.md): submitting patches to [docs.projectcalico.org](docs.projectcalico.org)
```
Comment (ozdanborne) [inline README.md]: this also needs `https://`

**Comment by ozdanborne** [comment]
@caseydavenport thanks for sticking with this and getting it in. Two nits, then ping me and ill merge

**Pull request merged**